src/components/BackgroundVideo/BackgroundVideo.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { s } from "./";
+
+export default function BackgroundVideo(props){
+
+	const {
+		videoSrc,
+		posterSrc = {}
```

Comment:
It's inside of an attribute called `video`, so `props.video.videoSrc` seems uneccesary? I reckon something like

```
const props = {
    sources: {
        mp4: "path/to/video.mp4",
        webm: "path/to/video.webm"
    },
    poster: "path/to/poster.jpg"
};
```